telescope/query.py

```diff
@@ -135,7 +135,7 @@ def _create_select_clauses(metric):
 class BigQueryQueryGenerator(object):
 
     database_name = 'plx.google'
-    table_format = '[{database_name}:m_lab.{table_date}.all]'
+    table_format = '{database_name}:m_lab.ndt.all'
```

Comment:
Oh neat! I don't know why we always did surrounding square brackets when it turns out they're not necessary. I guess nobody ever tried without them.